api/v1alpha1/storageconsumer_types.go

```diff
@@ -72,6 +72,10 @@ type ClientStatus struct {
 	PlatformVersion string `json:"platformVersion"`
 	// StorageClient Operator Version
 	OperatorVersion string `json:"operatorVersion"`
+	// ClusterID is the id of the openshift cluster
+	ClusterID string `json:"clusterId"`
+	// NamespacedName is the name and namespace of the StorageClient
+	NamespacedName string `json:"namespacedName"`
```

Comment:
1. There are several kinds of cluster identifiers. Extracting a _possibly_ embedded one from a `StorageConsumer`'s name instead of just having it as a standalone field seems like an unnecessary detour. 
2. It's already been decided. The dynamic fields you mentioned are reported as part of heartbeat mechanism. If you take a closer look, the above static fields are added **only** during the creation of `StorageConsumer`, and not via its update function (which is the actual "refresh").

Reply:
@leelavg We are not considering naming conventions as source a of truth. We used the cluster ID in the name because it allowed us to easily create a distinct name for each client. But now we need  the cluster id as a piece of information so we should provide it explicitly, exactly as @raaizik did

Reply:
@leelavg Can this be resolved?

Reply:
I asked for extracting the id due to it's nature being in constant throughtout the lifecycle of storageconsumer as there's no other construct which provides such durability of the value.

> If you take a closer look, the above static fields are added only during the creation of StorageConsumer, and not via its update function (which is the actual "refresh").
- pls excuse, the answer itself surfaces the issue here, ie, `added only during the creation`, however as it is tracked in another thread, closing this.

Reply:
Can be resolved.